src/main/java/seedu/address/logic/commands/commandAssign/AssignCommandFactory.java

```diff
@@ -30,6 +37,11 @@ else if ( (prefixes[0].equals(PREFIX_COURSEID) && prefixes[1].equals(PREFIX_STUD
                 outputCommand = new AssignStudentToCourseCommand(assignDescriptor);
             }
         }
+        else if ( (prefixes[0].equals(PREFIX_COURSEID) && prefixes[1].equals(PREFIX_ASSIGNMENTID)) || (prefixes[1].equals(PREFIX_COURSEID) && prefixes[0].equals(PREFIX_ASSIGNMENTID)) ){
```

Comment:
why do we need this check if the below line check isValidDescriptor already?

Reply:
good point have removed